acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/proxy/PassThroughAcceptanceTest.java

```diff
@@ -48,7 +49,7 @@ public void ethGetTransactionCountReturnCorrectNumber() {
   }
 
   @Test
-  public void ethBalanceRequestReturnsCorrectBalance() {
+  public void ethBalanceRequestReturnsCorrectBalance() throws IOException {
     final BigInteger ethSignerStartBalance = ethSigner().accounts().balance(RECIPIENT);
```

Comment:
Don't add `throws IOException`, deal with it at the lower level: consistency

Reply:
these were no longer necessary! removed them.